src/transformState.js

```diff
@@ -5,7 +5,25 @@
  * @param {Object[]} actions
  */
 function transformState(state, actions) {
-  // write code here
+  actions.forEach((action) => {
+    switch (action.type) {
+      case 'addProperties':
+        Object.assign(state, action.extraData);
```

Comment:
this can also be achieved with [spread operator ](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Spread_syntax) which is more commong nowadays

Reply:
unfortunately spread operator is difficult to use since linter prohibits reasigning state (as function input) and coach used this Object.assign method in today's lesson (live coding) aswell, so I decided to leave it that way